shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -548,6 +548,7 @@ - (void)hideTextInput {
 
 - (void)setTextInputClient:(int)client withConfiguration:(NSDictionary*)configuration {
   _view.keyboardType = ToUIKeyboardType(configuration[@"inputType"]);
+  _view.secureTextEntry = [@"true" isEqualToString:configuration[@"obscureText"]];
```

Comment:
Updated this PR to account for those plus ensure we fall back to a reasonable default if we don't get the json at all.